src/entities/Profile.ts

```diff
@@ -71,16 +72,22 @@ export default class Profile {
 	@IsOptional()
 	public twitter?: string;
 
+	@Column({ nullable: true })
+	@Matches(/https?:\/\/(www\.)?linkedin.com\/in\/(\w-?)+|(^$)/, { message: 'Not a valid linkedin URL' })
```

Comment:
Just some fixes for the RegEx (make http/https at the start optional, assert that the start of the string is tested with `^`)

Can you also set a max length on the string? You can use the class-validator decorator for this, or you could do it in RegEx. It's just so people can't abuse the API with really long, invalid LinkedIn URLs and store garbage in the database. I think the validation for the other social media prevents this.